apps/desktop/src/platform/main/autofill/native-autofill.main.ts

```
@@ -60,6 +60,18 @@ export class NativeAutofillMain {
           request,
         });
       },
+      // AssertionWihtoutUserInterfaceCallback
+      (error, clientId, sequenceNumber, request) => {
+        if (error) {
+          this.logService.error("autofill.IpcServer.assertion", error);
+          return;
+        }
```

question: shouldn't we return this error to the extension somehow?

---

We could add `this.ipcServer.completeError(clientId, sequenceNumber, error.message);` before the return?

---

Something like that eyah

---

I don't know what behaviour that causes, and I don't know how to trigger this error path to be honest.

---

Resolved in https://github.com/bitwarden/clients/pull/13051/commits/f15f5c6e166d173a35d013dc8f1d6f156b05acd6